### DISCUSSION TITLE
Using Clerk with server components in NextJS 13

### DISCUSSION THREAD

**Discussion by raqibhayder** [clerk/javascript#899] [Support request - DO NOT OPEN HERE] [answered]
We are using NextJS 13 (`app` directory) with Supabase. Is there a way to use Clerk with server components? I would appreciate if anyone can point me to an example repo or docs where Clerk is integrated with server components!

**Comment by dmccloud** [comment, marked as answer]
@raqibhayder have you had a look at these? https://clerk-docs.clerkpreview.com/quickstarts/nextjs/app-router

**Comment by raqibhayder** [comment]
Nice. Thank you for sharing.
